qiskit/_compiler.py

```diff
@@ -279,6 +293,57 @@ def load_unroll_qasm_file(filename, basis_gates='u1,u2,u3,cx,id'):
     return circuit_unrolled
 
 
+def best_subset(backend, n_qubits):
+    """Computes the qubit mapping with the best
+    connectivity.
+
+    Parameters:
+        backend (Qiskit.BaseBackend): A QISKit backend instance.
+        n_qubits (int): Number of subset qubits to consider.
+
+    Returns:
+        ndarray: Array of qubits to use for best
+                connectivity mapping.
+
+    Raises:
+        ValueError: Wrong number of qubits given.
+    """
+    device_qubits = backend.configuration['n_qubits']
+    cmap = np.asarray(backend.configuration['coupling_map'])
+
+    if n_qubits > device_qubits:
+        raise ValueError('Number of qubits greater than device.')
+    elif n_qubits <= 0:
+        raise ValueError('Number of qubits <= 0.')
+    if n_qubits == 1:
+        return np.array([0])
+
+    data = np.ones_like(cmap[:, 0])
+    sp_cmap = sp.coo_matrix((data, (cmap[:, 0], cmap[:, 1])),
+                            shape=(device_qubits, device_qubits)).tocsr()
+    best = 0
+    best_map = None
+    # do bfs with each node as starting point
+    for kk in range(sp_cmap.shape[0]):
+        bfs = cs.breadth_first_order(sp_cmap, i_start=kk, directed=False,
+                                     return_predecessors=False)
+
+        con = 0
+        for ii in range(n_qubits):
+            ss = bfs[ii]
```

Comment:
Can we have a better naming for `ss`? `bfs_node`? (to distinguish from the `node` of the sparse matrix index)

Reply:
Done.